content/integrations/entrust-keycontrol.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Entrust KeyControl
+date: 2023-08-01
+lastmod: :git
+draft: false
+tableOfContents: true
+---
+
+[Entrust KeyControl](https://www.entrust.com/digital-security/key-management/keycontrol) is a proprietary KMS that provides a secret store that can be used by KES.
+
+```goat
+                +-----------------------------------------+
+ .----------.   |   .----------.    .------------------.  |
+| KES Client +--+--+ KES Server +--+ Entrust KeyControl | |
+ '----------'   |   '----------'    '------------------'  |
+                +-----------------------------------------+
+```
+
+## Prerequisites
+
+
+This guide assumes that you have setup an Entrust KeyControl v10.1 cluster. 
+
+For instructions to set up an Entrust KeyControl cluster, refer to the Entrust documentation:
+ - [Entrust KeyControl on AWS](https://docs.hytrust.com/DataControl/10.1/Online/Content/Books/Amazon-Web-Services/Creating-KC-Cluster-AWS/Configuring-the-First-KC-Node-AWS.html)
+ - [Entrust KeyControl on prem](https://docs.hytrust.com/DataControl/10.1/Online/Content/OLH-Files/Help-content-map-all-books.html) 
+
+## Set up Entrust KeyControl
+
+Before setting up the KES Server, complete the following sections in KeyControl to add a new Vault, a Box, and one or more Users.
+
+### Create a new Vault
+
+1. Login into your KeyControl cluster as `secroot` and create a new **PASM** Vault.
```

Comment:
Is this the only possible login?

If it just requires root access, maybe go ambiguous as "Log in as a user with root-level permissions (``secroot``) and create a new **PASM**-type Vault.

Reply:
Tweaked